packages/default/scss/dock-manager/_layout.scss

```diff
@@ -0,0 +1,231 @@
+            border-inline-width: $kendo-dock-manager-splitbar-border-width;
+            border-inline-style: $kendo-dock-manager-splitbar-border-style;
+        }
+
+        .k-splitbar-vertical {
+            border-block-width: $kendo-dock-manager-splitbar-border-width;
+            border-block-style: $kendo-dock-manager-splitbar-border-style;
+        }
+
+        .k-tabstrip-content,
+        .k-splitter {
+            border-width: 0;
+        }
+
+        .k-toolbar .k-separator-horizontal {
+            width:  $kendo-line-height-em;
+            height: 0;
+            border-width: $kendo-dock-manager-separator-border-width 0 0;
+        }
+
+    }
+
+        // Needed because of a bug in Chrome - Issue 1473554
+        // A fix will be shipped likely in milestone 118 and then we can revisit this
+        @supports  (not (-ms-ime-mode: none)) and (not (overflow: -webkit-marquee)) and (not (-moz-appearance:none)) {
+            .k-dock-manager-toolbar .k-toolbar-button {
+                padding-block: $kendo-dock-manager-toolbar-button-padding-y;
+                padding-inline: $kendo-dock-manager-toolbar-button-padding-x;
+            }
+            .k-dock-manager-toolbar .k-toolbar-button .k-button-text {
+                writing-mode: vertical-lr;
+            }
+        }
+
+    .k-pane-header {
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        padding-inline: $kendo-dock-manager-pane-header-padding-x;
+        border-block-end-width: $kendo-dock-manager-pane-header-border-width;
+        border-block-end-style: $kendo-dock-manager-pane-header-border-style;
+    }
+
+    .k-pane-title {
+        flex: 1 1 auto;
+        padding-block: $kendo-dock-manager-pane-title-padding-y;
+        font-family: $kendo-font-family;
+        font-size: $kendo-dock-manager-pane-title-font-size;
+        line-height: $kendo-dock-manager-pane-title-line-height;
+        font-weight: $kendo-dock-manager-pane-title-font-weight;
+        text-overflow: ellipsis;
+        overflow: hidden;
+        cursor: default;
+        display: flex;
+        flex-flow: row nowrap;
+    }
+
+    .k-pane-content {
+        padding-block: $kendo-dock-manager-pane-content-padding-y;
+        padding-inline: $kendo-dock-manager-pane-content-padding-x;
+        text-overflow: ellipsis;
+        overflow: hidden;
+        flex: 1;
+    }
+
+    .k-pane-actions {
+        display: flex;
+        flex-flow: row nowrap;
+        align-items: center;
+        justify-content: center;
+    }
+
+    .k-pane .k-pane-tabbed {
+        padding-block-start: $kendo-dock-manager-tabbed-pane-padding-y;
+    }
+
+    .k-dock-manager-toolbar {
+        writing-mode: vertical-lr;
+        border-color: inherit;
+        box-shadow: none;
+        flex-shrink: 0;
+
+        &::before {
+            height: 0;
+        }
+    }
+
+    .k-toolbar-left {
+        border-width: 0 $kendo-dock-manager-border-width 0 0;
+    }
+
```

Comment:
probably address this with https://github.com/telerik/kendo-themes/pull/4300/commits/ce7b79b7ea0465420200f2abdeb3f682c9d713f6